src/Search.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
+import SearchButton from "./SearchButton"
 
-const Search = () => {
+const Search = (props) => {
```

Comment:
It's a little strange, but not wrong, to take props and then not use them - in your other components where you don't use props, you just used `()` as the argument list :)